src/java.desktop/share/classes/java/beans/beancontext/BeanContextSupport.java

```diff
@@ -305,8 +305,8 @@ public void    remove()  { /* do nothing */      }
      */
 
     /**
-     * protected nested class containing per child information
-     * in the "children" hashtable.
+     * A protected nested class containing per-child information
+     * in the {@code children} hashtable.
      */
     protected class BCSChild implements Serializable {
 
```

Comment:
Shall we indent `serialVersionUID`? It's for `BCSChild` but its indentation suggests it's for the surrounding class.

Reply:
> Shall we indent `serialVersionUID`? It's for `BCSChild` but its indentation suggests it's for the surrounding class.

It's right below the class declaration.